src/rgw/rgw_admin.cc

```diff
@@ -158,6 +159,9 @@ void _usage()
   cout << "  orphans find               init and run search for leaked rados objects (use job-id, pool)\n";
   cout << "  orphans finish             clean up search for leaked rados objects\n";
   cout << "  orphans list-jobs          list the current job-ids for orphans search\n";
+  cout << "  role create                create a role\n";
+  cout << "  role delete                delete a role\n";
+  cout << "  role get                   get a role\n";
```

Comment:
can we add a bit more context? something like `create an AWS role for use with STS`? `delete a role` and `get a role` are fine as they are